src/chainlit/frontend/src/state/element.ts

```diff
@@ -42,6 +49,12 @@ export interface IPdfElement extends IElement {
   url?: string;
 }
 
+export interface IAudioElement extends IElement {
+  type: 'audio';
+  content?: string;
```

Comment:
I don't think the content should be of type string, maybe you copy pasted from the TextElement?

Reply:
fixed